charts/model-engine/values.yaml

```diff
@@ -1,11 +1,313 @@
-dd_trace_enabled: true
-spellbook:
-  enabled: false
-redis:
-  auth:
+# If specified, will override the name of the deployed services
+# Otherwise, defaults to the Chart name, typically "model-engine"
+# serviceIdentifier:
+
+# The Kubernetes cluster name in which the Model Engine is deployed
+clusterName:
+
+secrets:
+  # Either of the below AWS secrets expect a key named "database_url" with a fully specified database URL including
+  # the username and password.
+
+  # Use the Cloud database secret name to pull from AWS Secrets Manager
+  # cloudDatabaseSecretName:
+  # Use the Kubernetes database secret name to pull from Kubernetes Secrets
+  kubernetesDatabaseSecretName:
+
+  # This secret must have a fully specified database URL including the password (auth token)
+  # It should be under "cache_url" key in the secret
+  # redisAwsSecretName:
+  # Kubernetes secret containing a key `auth_token` that contains the redis auth token for connection
+  # Will not be used if `redisAwsSecretName` is set. Used in conjunction with REDIS_HOST and REDIS_PORT env vars
+  kubernetesRedisSecretName:
+
 db:
+  # Runs an initial database schema migration on deployment if set to true
   runDbInitScript: false
-balloonNodeSelector:
-  node-lifecycle: normal
-nodeSelector:
-  node-lifecycle: normal
+
+replicaCount:
+  # The gateway service is the entrypoint for all requests to the Model Engine
+  gateway: 1
+  # The cacher service is responsible for caching kubernetes API requests
+  cacher: 1
+  # The builder service is responsible for creating new deployments and other kubernetes resources
+  builder: 1
+
+# Tag of the model engine image that will be used for the model engine deployments
+tag:
+# Sets the 'env' label on the pods and primarily used for metadata tagging
+context:
+# Specifies core services' image repositories
+image:
+  gatewayRepository: public.ecr.aws/b2z8n5q1/model-engine
+  builderRepository: public.ecr.aws/b2z8n5q1/model-engine
+  cacherRepository: public.ecr.aws/b2z8n5q1/model-engine
+  forwarderRepository: public.ecr.aws/b2z8n5q1/model-engine
+  pullPolicy: Always
+
+# Specifiers for the core model engine service deployments
+nodeSelector: { }
+tolerations: [ ]
+affinity: { }
+
+# Specifies the configuration on the Gateway service kube service
+service:
+  type: ClusterIP
+  port: 80
+
+# Creates istio virtual services for the Model Engine using the global domain nome and gateway specified below
+virtualService:
```

Comment:
i think this is a backwards incompatible change; can we leave this as virtualservice